main.cpp

```diff
@@ -27,6 +28,8 @@ void finish(GameData& gameData) {
 int main() {
   start();
 
+  srand(time(nullptr));
```

Comment:
Modern c++ random source is recommended https://en.cppreference.com/w/cpp/header/random